### ISSUE TITLE
There should be a warning on this integration page

### ISSUE THREAD

**Issue by MiNi33** [home-assistant/home-assistant.github.io#35166]
### Feedback

Growatt is apparently failing with ‘openapi’. While I am logged in I receive 0 on every sensor in the integration, messing up my Energy dashboard.
If their service is so unreliable it is not wise to use this integration for your Energy dashboard!

### URL

https://www.home-assistant.io/integrations/growatt_server/

### Version

2024.10.1

**Comment by SanderKo85** [comment]
indeed, same here, logged in, 0 information/data received. HA dashboard is empty since the beginning of October.
